templates/base_index.html

```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
+                alt="CIS logo",
+                width="90",
+                height="90",
+                hi_def=True,
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </li>
+          <li class="p-inline-images__item">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/44b52356-csec_logo.svg",
+                alt="CSEC logo",
+                width="79",
+                height="90",
+                hi_def=True,
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </li>
+          <li class="p-inline-images__item">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/49806a3f-IEC_logo.svg",
+                alt="IEC logo",
+                width="130",
+                height="40",
+                hi_def=True,
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </li>
+          <li class="p-inline-images__item">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/5a0299dd-ISO_logo.svg",
+                alt="ISO logo",
+                width="130",
+                height="45",
+                hi_def=True,
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="p-strip--light">
+      <div class="u-fixed-width">
+        {{
+          image(
+            url="https://assets.ubuntu.com/v1/3ec2dd61-Money.svg",
+            alt="",
+            width="128",
+            height="128",
+            hi_def=True,
+          ) | safe
+        }}
+        <h2>Significant enterprise savings</h2>
+        <h5>Sustainable open source at the lowest cost, everywhere</h5>
+        <p>Rethink what’s possible with Linux and open source.</p>
+        <p>Companies engage Canonical to drive down open source operating costs. Automate everything: multi-cloud operations, bare metal provisioning, edge clusters and IoT.</p>
+        <ul class="p-list is-split">
+          <li class="is-ticked">No mandatory subscriptions for Ubuntu</li>
+          <li class="is-ticked">Security, support and fully managed offerings</li>
+          <li class="is-ticked">Public cloud, data centre, edge cluster & appliances</li>
+          <li class="is-ticked">Long term maintenance commitment</li>
+          <li class="is-ticked">Transparent pricing for enterprise and ISV</li>
+          <li class="is-ticked"><a href="/engage/vmware-to-charmed-openstack">Switch from VMware to OpenStack</a></li>
+          <li class="is-ticked"><a href="/tco-calculator">TCO analysis of Open Infra</a></li>
+          <li class="is-ticked"><a href="/engage/cloud-economics">451 study on private/public cloud costs</a></li>
+        </ul>
+        <a href="#get-in-touch">Let’s talk open source ></a>
```

Comment:
```suggestion
        <a href="/contact-us" class="js-invoke-modal">Let’s talk open source&nbsp;&rsaquo;</a>
```

Reply:
The reason for this is that `#get-in-touch` requires JS to trigger the modal contact form so if someone doesn't have js they will link to the contact-us page but if they do the modal will takeover.